pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1557,18 +1558,46 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
     return JSON.parse(stdout).SPNetworkDataType;
   }
 
+  protected async installContainerdShims() {
+    // shims bundled with the app itself
+    const bundleDir = path.join(paths.resources, 'linux', 'internal');
+    // shims managed by the user
+    const shimDir = path.join(paths.appHome, 'containerd-shims');
+
+    await this.execCommand('mkdir', '-p', 'bin');
```

Comment:
That's implicitly `/home/<user>/bin`?

Reply:
It is. I copied this from `installK3s`, and was wondering about the same thing first. How can we improve this?

Reply:
I'm not using `/root` instead of `$HOME/bin`, which didn't work right with WSL.